small/matras.c

```diff
@@ -364,6 +364,34 @@ matras_touch_reserve(struct matras *m, int count)
 	return matras_allocator_reserve(m->allocator, max_extents_required);
 }
 
+#define MATRAS_DIV_ROUND_UP(a, b) ((a) + (b) - 1) / (b)
```

Comment:
There's `small/util.h` Let's define it there.

Reply:
It wasn't dropped. There are still two commits.

Reply:
Yes, that was some strange bug, amended the same commit and repushed, so now it's a single commit.